op-e2e/custom_gas_token_test.go

```diff
@@ -278,6 +410,7 @@ func TestCustomGasToken(t *testing.T) {
 	checkL1TokenNameAndSymbol(t, enabled)
```

Comment:
To debug this easily, you can wrap all the other tests in an:

```
if true {
...
}
```

branch, because its golang it wont compile if you comment it out, then you can do `go test -run TestCustomGasToken -v` to run the tests locally. Make sure to `rm -rf .devnet` and `make devnet-allocs`